rust/src/mask/config/mod.rs

```diff
@@ -204,6 +208,7 @@ impl MaskConfig {
         }
     }
 
+    /// Gets the finite group order value for masking/unmasking.
```

Comment:
i'm a bit hesitant on this matter. there is always the question on wether 1) the docstring explains only how this method is used for what, or 2) the docstring also explains the details on how and why this works as it does. i left out the mathematical details on the latter and included a more detailed documentation on the former in the module docs.

also we want to change this catalogue, which is quite limiting in the choices it provides, to a computable config which then covers all cases in the future. this will make a docstring on the current details obsolete.

Reply:
i agree the module-level docs do a great job of describing usage at a high-level. perhaps it's just that the use of "magic numbers" always makes me a bit uncomfortable. But i take your point that it can be tackled later when we have it more generic.